vtex/intl-equalizer
# Add ICU message parser and fail if syntax errors exists

#### What is the purpose of this pull request?

Adds the `intl-formatmessage-parser` to parse ICU messages and check for syntax errors.

#### What problem is this solving?

This will allow us to review localizaion and translation PRs more easily because we won't have to check manually anymore for syntax erros in the message files.

#### How should this be manually tested?

Build with `yarn build` and run `node ../path/to/repo/dist/index.js` inside your app to check for the errors.

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/10223856/95501568-e22c1080-097e-11eb-9197-059e983bbf0e.png)

#### Types of changes
